presto-main/src/test/java/com/facebook/presto/metadata/TestSignature.java

```diff
@@ -135,6 +140,62 @@ public void testBindUnknown()
     }
 
     @Test
+    public void testBindBigintToDecimal()
+            throws Exception
+    {
+        assertFunctionBind(
```

Comment:
I know. It is quite hard to understand which parameter stands for what. But it is better to refactor it in a separate PR.